tutorials/export/exporting_pcks.rst

```diff
@@ -51,6 +60,23 @@ PCK files usually contain, but are not limited to:
 The PCK files can even be an entirely different Godot project, which the
 original game loads in at runtime.
 
+.. _doc_exporting_pcks_about_mods_using_pck_files:
+
+About mods using PCK files
```

Comment:
Usually headers are are either in the form "Using X for Y" or "Doing X with Y" or sometimes just "X", where X is the name of the feature. One of these suggestions is more idiomatic:
```suggestion
Using PCK files for modding
```
or
```suggestion
Using PCK files for mods
```
or
```suggestion
Modding games with PCK files
```
or
```suggestion
Supporting mods with PCK files
```
We should wait for other docs maintainers to comment before changing this. Note that the anchor and links to this section need to change, too.